spec/system/candidate_interface/candidate_needs_to_provide_two_new_referees_spec.rb

```diff
@@ -4,14 +4,15 @@
   include CandidateHelper
 
   scenario "Candidate provides a new referee because 2 didn't respond" do
+    FeatureFlag.activate('pilot_open')
     FeatureFlag.activate('show_new_referee_needed')
 
     given_i_am_signed_in_as_a_candidate
     and_i_have_submitted_my_application
     and_both_of_my_referees_havent_responded_within_a_reasonable_timeframe
 
-    when_i_visit_the_application_dashboard
-    then_i_see_that_i_need_new_references
+    when_i_visit_the_start_page
+    then_i_see_the_interstitial_page_to_add_new_referees
```

Comment:
Can we keep the existing tests as well? We still need to make sure we show the warning on the dashboard.